g11n-ws/modules/md-data-api-s3impl/src/main/java/com/vmware/vip/messages/data/conf/S3Config.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.messages.data.conf;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+/**
+ * the configuration of the S3 client
+ */
+@Configuration
+public class S3Config {
```

Comment:
where is the properties file managed by SpringBoot?

Reply:
when done the confidential information feature，all the configuration managed by Spring Boot after open source